python/pynessie/client/nessie_client.py

```diff
@@ -263,7 +260,8 @@ def merge(
         merge_json = MergeSchema().dump(Merge(from_ref, str(from_hash)))
         merge(self._base_url, self._auth, onto_branch, merge_json, old_hash, self._ssl_verify)
 
-    def cherry_pick(self: "NessieClient", branch: str, from_ref: str, old_hash: Optional[str] = None, *hashes: str) -> None:
+    # pylint: disable=keyword-arg-before-vararg
```

Comment:
Should we disable this as well from the config file?

Reply:
it was disabled before in the config file but this PR removes it there. due to backwards compatibility we have to keep it here but ignore it.

Reply:
same as for `commit` method above